jctools-core/src/main/java/org/jctools/queues/MpUnboundedXaddArrayQueue.java

```diff
@@ -321,7 +311,7 @@ private R newOrPooledChunk(R prevChunk, long nextChunkIndex)
     }
 
 
-    R pollNextBuffer(R cChunk, long cIndex)
+    protected final R pollNextBuffer(R cChunk, long cIndex)
```

Comment:
`final` is fine, but `protected` is public API for anyone extending the queue

Reply:
So we don't want the subclasses of users to call this, got it, let me change it :+1

Reply:
public API -> liability